lib/eventboss/topic.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Eventboss
+  class Topic
+    class << self
+      def build_arn(event_name:, source_app: nil)
+        [
+          'arn:aws:sns',
+          Eventboss.configuration.eventboss_region,
+          Eventboss.configuration.eventboss_account_id,
+          build_name(
+            event_name: event_name,
+            source_app: source_app
+          )
+        ].join(':')
```

Comment:
There is a competing implementation on how to build ARN from params in `Queue` class, maybe it can be put in one place?

Reply:
It would require creating additional entity (like a module) that would keep only this logic. I could do this but not sure if it's not an overkill... Wdyt?

Reply:
Yeah, maybe. We can refactor in the future (don't want to overcomplicate this)